src/lib/adservices.ts

```
@@ -0,0 +1,23 @@
+import { Alert, Platform } from 'react-native'
```

ESLint

[error] 'Alert' is defined but never used.

(@typescript-eslint/no-unused-vars)